torch/utils/benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -28,7 +28,9 @@
     CompletedProcessType = subprocess.CompletedProcess
 
 
-FunctionCount = NamedTuple("FunctionCount", [("count", int), ("function", str)])
+class FunctionCount(NamedTuple):
+    count: int
```

Comment:
This is redefining the built in count in tuple(). Need

```suggestion
    count: int  # type: ignore[assignment]
```

Reply:
Happy to if that’s what people want! I am not familiar with this part of the code so may break things. Is this a public API?

Reply:
Created https://github.com/pytorch/pytorch/issues/105471 for now